examples/viewer.py

```diff
@@ -53,10 +53,22 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
         )
 
         # Compute environment camera resolution based on the number of environments to render in the window.
-        surface_size: tuple(int, int) = (
+        window_size: tuple(int, int) = (
             self.sim_settings["window_width"],
             self.sim_settings["window_height"],
         )
+
+        configuration = self.Configuration()
+        configuration.title = "Habitat Sim Interactive Viewer"
+        configuration.size = window_size
+        Application.__init__(self, configuration)
+        self.fps: float = 60.0
+
+        # Compute environment camera resolution based on the number of environments to render in the window.
+        surface_size: tuple(int, int) = (
+            mn.gl.default_framebuffer.viewport.size()[0],
+            mn.gl.default_framebuffer.viewport.size()[1],
+        )
```

Comment:
```suggestion
        surface_size: mn.Vector2i = mn.gl.default_framebuffer.viewport.size()
```

The Configuration accepts a `Vector2i`, so going through a tuple is unnecessary extra work. Alternatively you could use `self.framebuffer_size`, which is shorter, and returns the same (while `window_size` returns "virtual pixels"):

```py
surface_size: mn.Vector2i = self.framebuffer_size
```